content/en/contribute/code/core/dev-environment.md

```diff
@@ -137,6 +139,45 @@ Now you can start CouchDB. The login for your CHT instance will be `medic` and t
 cd ~/cht-docker 
 COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose -f docker-compose.yml -f couchdb-override.yml up -d
 ```
+#### CouchDB Setup in 4.x for Apple Silicon
+
+You will need to build the Docker images locally on your machine because the images hosted publicly do not currently support Apple's ARM architecture. While in the root directory of your `cht-core` project:
+
+```shell
+npm run local-images
+```
+
+After the `npm` command completes successfully, a `local-build` folder will be created in the root directory of your `cht` project.
```

Comment:
can we also confirm that the folder has three files cht-core.yml, cht-couchdb.yml & cht-couchdb-clustered.yml

Reply:
yes, that's a good suggestion.